src/test/resources/features/Meeds/MeedsSpaces.feature

```diff
@@ -75,8 +75,6 @@ Feature: Meeds Space
     Then I check that space application installer drawer is displayed
     And I check that applications are displayed
 
-    When I click to add application 'Wallet'
-    Then I check that application 'Wallet' is added to applications page
```

Comment:
without those lines, the test case become useless IMO

Reply:
In builder when creating spaces ,  Wallet application is already among the application added , I have removed those line because when opening the space application installer drawer ,it is empty there is no application to add .

Reply:
Yep, so we may have to add a dedicated application to test on it or we can disable mandatory on Wallet to test on it as well.

Reply:
I did the second preposition by removing then adding Notes application as there is no "Remove" option for the admin User for wallet application .